<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>97</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>LED ON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>97</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>LED OFF</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>181</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>97</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Check</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>230</y>
     <width>97</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
